cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/node-labeler/clusterrolebinding.yaml

```diff
@@ -0,0 +1,11 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+    name: node-labeler
+roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: node-labeler
+subjects:
+    - kind: ServiceAccount
```

Comment:
need a namespace for this serviceaccount

Reply:
It gets added by kustomize.